src/Test/Hspec/Hedgehog.hs

```diff
@@ -146,6 +146,9 @@ instance Example (PropertyT IO ()) where
     type Arg (PropertyT IO ()) = ()
     evaluateExample e = evaluateExample (\() -> e)
 
+propertyWithoutLocation :: PropertyT IO () -> Property
+propertyWithoutLocation prop = withFrozenCallStack $ property prop
```

Comment:
I think this will freeze the callstack provided in `prop` as well, which may not be what we want - `withFrozenCallStack property prop` will only freeze the callstack as far as `property` is concerned

Reply:
`prop` does not have a `HasCallStack` constraint, so **I would be very surprised if it makes a difference**.

However, removing the dollar will give us the exact same result, but it is a little bit shorter. So I am OK with that.

Reply:
We can make it even shorter by making it point free. Again, no semantic difference, but it won't hurt.